build.gradle

```diff
@@ -1,29 +1,41 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.10"
+    ext.versions = [
+            'kotlin': '1.5.21'
+    ]
+    ext.deps = [
+            // Kotlin
+            'kotlinStdLib'        : "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}",
```

Comment:
Nitpick but do we want to standardize using double or single quotes?

Reply:
Unfortunately groovy requires the use of double quotes when using string interpolation 😞  I prefer using single quotes, but cool with updating all to double quotes to keep things uniform.

Reply:
I don't feel that strongly - we can make the convention double for string interpolation and single for everything else or go all double.

Reply:
I'm cool going all double-quotes. If we need to use string interpolation later for certain libraries, we won't need to update the type of quotes used.